src/core/include/openvino/op/clamp.hpp

```diff
@@ -48,6 +48,8 @@ class OPENVINO_API Clamp : public Op {
     }
     OPENVINO_SUPPRESS_DEPRECATED_START
     bool evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const override;
+    bool evaluate_lower(const HostTensorVector& outputs) const override;
+    bool evaluate_upper(const HostTensorVector& outputs) const override;
```

Comment:
why not to override new methods?

Reply:
Agree, can we use new API?

Reply:
let's just plan to do it for all the `evaluate_lower/upper` methods? 
Created a ticket for that: 95057